test/shared/phpunit_tap.xml

```diff
@@ -1,6 +1,8 @@
 <phpunit
   colors ="true"
   verbose="true"
+  bootstrap="test/bootstrap.php"
+  cacheResult="false"
   printerClass="PHPUnit_Util_Log_TAP"
```

Comment:
Does this class exist in used phpunit versions?

Reply:
Sure, it does not. I'll drop the file.